xc/device.ml

```diff
@@ -1863,8 +1863,12 @@ module Dm_Common = struct
     | VNC of disp_intf_opt * string option * bool * int * string option (* IP address, auto-allocate, port if previous false, keymap *)
     | SDL of disp_intf_opt * string (* X11 display *)
 
-  type media = Disk | Cdrom
-  let string_of_media = function Disk -> "disk" | Cdrom -> "cdrom"
+  type media = Disk | Cdrom | Floppy
```

Comment:
I understand that this is an incremental change but I would have liked to move this into a module to regain some structure:

```
module Media = struct
  type t = Disk | Cdrom | Floppy

  let to_string = function
    | Disk -> "disk" ..
   ..

  ..
end
```